engine/apps/alerts/escalation_snapshot/escalation_snapshot_mixin.py

```diff
@@ -253,6 +253,8 @@ def start_escalation_if_needed(self, countdown=ScenarioStep.CROSS_ACTION_DELAY,
 
         logger.debug(f"Start escalation for alert group with pk: {self.pk}")
 
+        # TODO: allow to start escalation at specific step?
```

Comment:
Do we plan to implement it as a part of this PR?

Reply:
No, will leave this for later, will remove the comment for now.